lib/bson.js

```diff
@@ -1,65 +1,11 @@
 'use strict';
 
-var mongodb = require('mongodb');
-var vm      = require('vm');
-var json    = require('./json');
+const parser = require('mongodb-query-parser');
+const ESJON = require('mongodb-extended-json');
```

Comment:
https://github.com/mongodb-js/mongodb-extjson#note-this-libray-has-been-merged-into-the-4x-version-of-the-bson-package-and-is-no-longer-being-maintained-as-a-standalone-package

https://github.com/mongodb/js-bson#ejsonparsetext-options

Reply:
Unfortunately, can't update bson until mongodb-query-parser is updated to a more recent version.